Ochowo/Star-Wars
# filter by gender

#### What does this PR do?
- Filters the characters by gender

#### Description of the task to be completed?
- Set up component
- Create action and reducers
- Connect component to store

#### How to manually test
- Clone the repo
- Run `npm install`
- Run `npm run start:dev`
- Visit `http://localhost:3000/`
- Click on the dropdown to view the list of movies
- Click on each item on the list to view more information
- Select from the list of gender to see characters based on each selection

#### What are the relevant pivotal tracker stories?
- N/A

#### Any background context you want to add?
- N/A

#### Screenshots
- N/A
